modules/geo/build.gradle

```
@@ -28,9 +28,10 @@
  * under the License.
  */
 apply plugin: 'opensearch.yaml-rest-test'
+apply plugin: 'opensearch.internal-cluster-test'
 
 opensearchplugin {
-  description 'Placeholder plugin for geospatial features in OpenSearch. only registers geo_shape field mapper for now'
+  description 'Plugin for geospatial features in OpenSearch. Registering the geo_shape and aggregations GeoBounds on Geo_Shape and Geo_Point'
```

Do we need to add this to description?
```suggestion
  description 'Plugin for geospatial features in OpenSearch.'
```

---

I just updated the description as it was not correct. I want to provide exact information what would be provided from this plugin. "Plugin for geospatial features in OpenSearch." This statement is not 100% correct all alone. Hence added what other things will be added from this plugin for now.

Once we keep on moving more things and from Geo standpoint, everything becomes consolidated we can write it provides GeoSpatial feature.